cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -0,0 +1,438 @@
+        |* <<cypher-temporal-timezones, Time zones>>
+        |* <<cypher-temporal-instants, Temporal instants>>
+        | ** <<cypher-temporal-specifying-temporal-instants, Specifying temporal instants>>
+        |  *** <<cypher-temporal-specify-date, Specifying dates>>
+        |  *** <<cypher-temporal-specify-time, Specifying times>>
+        |  *** <<cypher-temporal-specify-time-zone, Specifying time zones>>
+        | ** <<cypher-temporal-accessing-components-temporal-instants, Accessing components of temporal instants>>
+        |* <<cypher-temporal-durations, Durations>>
+        | ** <<cypher-temporal-specifying-durations, Specifying durations>>
+        | ** <<cypher-temporal-accessing-components-durations, Accessing components of durations>>
+      """.stripMargin)
+    note{
+      p("""Refer to <<query-functions-temporal>> for information regarding temporal _functions_ allowing for the creation and manipulation of temporal values.""")
+      p("""Refer to <<query-operators-temporal>> for information regarding temporal _operators_.""")
+      p("""Refer to <<cypher-ordering>> for information regarding the comparison and ordering of temporal values.""")
+    }
+    section("Introduction", "cypher-temporal-introduction") {
+      p(
+        """The following table depicts the temporal value types and supported components:
+          |
+          |[options="header", cols="^,^,^,^", width="85%"]
+          ||===
+          || Type | Date support | Time support | Time zone support
+          || Date | X  | |
+          || Time | | X | X
+          || LocalTime | | X |
+          || DateTime |  X | X | X
+          || LocalDateTime | X | X |
+          || Duration | - | - | -
+          ||===
+          |
+          |""")
+      p(
+        """_Date_, _Time_, _LocalTime_, _DateTime_ and _LocalDateTime_ are _temporal instant_ types.
+          |A temporal instant value expresses a point in time -- an instant -- with varying degrees of precision.
+        """.stripMargin)
+      p(
+        """By contrast, _Duration_ is not a temporal instant type.
+          |A _Duration_ represents a temporal amount, capturing the difference in time between two instants, and can be negative.""".stripMargin)
+      important {
+        p("Duration only captures the amount of time between two instants, and thus does not encapsulate a start time and end time.")
+      }
+    }
+    section("Time zones", "cypher-temporal-timezones") {
+      p(
+        """Time zones are represented in one of two ways: either (i) as an offset from UTC, or (ii) as a logical identifier of a _named time zone_ (these are based on the https://www.iana.org/time-zones[IANA time zone database]).
+          |In either case the time is stored as UTC internally, and the time zone offset is only applied when the time is presented.
+          |This means that temporal instants can be ordered without taking time zone into account.""".stripMargin)
```

Comment:
This is not entirely true. If they are the same point in time, the time zone must be taken into account.